plone/namedfile/scaling.py

```diff
@@ -31,6 +31,11 @@
 import logging
 import six
 
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from io import BytesIO as StringIO
```

Comment:
I think you can safely use:
```
from io import BytesIO
```
in all Python versions

Reply:
It's like it is in 

https://github.com/plone/plone.scale/blob/master/plone/scale/scale.py#L9

I can change it anyways.

Reply:
Oh I see!
I also see that file has also the lines:
```
if isinstance(image, (bytes, str)):
     image = StringIO(image)
```
which is buggy on Python3:
```
>>> from io import BytesIO as StringIO
>>> StringIO("a")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: a bytes-like object is required, not 'str'
```
But that's another story, I would not focus on fixing that other bug, I will open another ticket for that.